.github/workflows/CI.yml

```diff
@@ -16,7 +16,11 @@ jobs:
         python -m pip install --upgrade pip
         pip install pytest
         pip install .[jax_cpu]
+        pip install .[pytorch_cpu]
+        pip install .[wmt]
     - name: Run pytest
       run: |
         pytest -vx tests/version_test.py
         pytest -vx tests/workloads/imagenet/imagenet_jax/workload_test.py
+        pytest -vx tests/test_num_params.py
+        pytest -vx tests/test_param_shapes.py
```

Comment:
Can we not just call something like `pytest -vx tests/`?
If we want to exclude certain tests it might be best to mark them as slow (e.g. https://docs.pytest.org/en/6.2.x/mark.html)

Reply:
Cool, I didn't know about this functionality. I think Zack wanted to exclude the slow tests from being run by GitHub workflows for now.